packages/components/pie-toast/src/defs.ts

```diff
@@ -78,4 +87,5 @@ export const defaultProps: DefaultProps = {
     isStrong: false,
     isDismissible: true,
     isMultiline: false,
+    duration: defaultDuration,
```

Comment:
We can probably delete the variable and assign the value directly here.
```suggestion
    duration: 5000,
```

Reply:
@xander-marjoram I wanted to avoid "magic numbers" that's why I kept the variable.